scripts/installer-lang/es.ps1

```diff
@@ -1,49 +1,49 @@
+    StopScript      = "El programa se ha detenido"
+    MsSpoti         = "Se ha detectado la versión de Spotify de Microsoft Store, que no es compatible"
+    MsSpoti2        = "¿Desea desinstalar la edición de Spotify de Microsoft Store? [Y/N]"
+    MsSpoti3        = "Desinstalando automáticamente Spotify de Microsoft Store..."
     MsSpoti4        = "Desinstalación de Spotify de Microsoft Store..."
     Prem            = "Modificación para la cuenta prémium..."
     OldV            = "Se ha encontrado una versión obsoleta de Spotify"
-    OldV2           = "Su versión de Spotify {0} está obsoleta, se recomienda actualizar a {1}"
-    OldV3           = "¿Quiere actualizar? [Y/N]"
+    OldV2           = "Su versión de Spotify {0} está obsoleta, se recomienda actualizar a la versión {1}"
+    OldV3           = "¿Desea actualizar? [Y/N]"
     AutoUpd         = "Actualización automática a la versión recomendada"
-    DelOrOver       = "¿Desea desinstalar la versión actual de {0} o instalar sobre ella? Y [Desinstalar] / N [Instalar encima]"
-    DelOld          = "Desinstalando el antiguo Spotify..."
-    NewV            = "Se ha encontrado una versión no compatible de Spotify"
-    NewV2           = "Su versión de Spotify {0} aún no se ha probado; actualmente la versión estable es la {1}"
-    NewV3           = "¿Quieres continuar con la versión {0} (posibles errores)? [Y/N]"
-    Recom           = "¿Desea instalar la versión recomendada {0}? [Y/N]"
-    DelNew          = "Desinstalando un Spotify no probado..."
+    DelOrOver       = "¿Desea desinstalar la versión actual ({0}) o sobreescribirla? Y [Desinstalar] / N [Sobreescribir]"
+    DelOld          = "Desinstalando la versión antigua de Spotify..."
+    NewV            = "Se ha encontrado una versión de Spotify no compatible"
+    NewV2           = "Su versión de Spotify ({0}) aún no se ha probado; la versión estable para SpotX es la {1}"
+    NewV3           = "¿Quieres continuar con la versión ({0}) (pueden surgir errores)? [Y/N]"
+    Recom           = "¿Desea instalar la versión recomendada ({0})? [Y/N]"
+    DelNew          = "Desinstalando una versión de Spotify no probada..."
     DownSpoti       = "Descargando e instalando Spotify"
     DownSpoti2      = "Espere un momento..."
-    PodcatsOff      = "Pódcast desactivados"
-    PodcastsOn      = "Pódcast activados"
-    PodcatsSelect   = "¿Quiere desactivar los pódcast, episodios y audiolibros de la página principal? [Y/N]"
-    DowngradeNote   = "Es recomendable bloquear las actualizaciones porque ha habido una bajada en la versión de Spotify"
+    PodcatsOff      = "Pódcasts desactivados"
+    PodcastsOn      = "Pódcasts activados"
+    PodcatsSelect   = "¿Quiere desactivar los pódcast, programas y audiolibros de la página principal? [Y/N]"
+    DowngradeNote   = "Se recomienda bloquear las actualizaciones porque se ha instalado una versión anterior de Spotify"
```

Comment:
the original now looks like this

"It is recommended to block because there is already a newer version of Spotify"